bin/install_RabbitMQ.sh

```diff
@@ -0,0 +1,8 @@
+cd $IIP_TMP_DIR
+curl -s https://packagecloud.io/install/repositories/rabbitmq/erlang/script.rpm.sh | bash
+yum install -y erlang
+rpm --import https://packagecloud.io/rabbitmq/rabbitmq-server/gpgkey
+rpm --import https://packagecloud.io/gpg.key
+curl -s https://packagecloud.io/install/repositories/rabbitmq/rabbitmq-server/script.rpm.sh | bash
```

Comment:
We probably want to separate this is 2 steps for clarity. No need for brevity here.